.circleci/config.yml

```diff
@@ -36,8 +36,26 @@ jobs:
       - run:
           name: UnitTests
           command: make ci-unit-test
+
+      # code coverage
+      - run:
+          name: Upload Code Coverage Report
+          command: |
+            curl -s https://codecov.io/bash > codecov;
+            VERSION=$(grep 'VERSION=\".*\"' codecov | cut -d'"' -f2);
```

Comment:
This should be `VERSION=$(grep 'VERSION=\"[0-9\.]*\"' codecov | cut -d'"' -f2);`